apps/app_rand.c

```diff
@@ -95,9 +95,9 @@ int app_RAND_write_file(const char *file)
 
     if (egdsocket || !seeded) {
         /*
-         * If we did not manage to read the seed file, we should not write a
-         * low-random seed file back -- it would suppress a crucial warning
-         * the next time we want to use it.
+         * If we didn't manage to read the seed file, don't write a
+         * file out -- it would suppress a crucial warning the next
```

Comment:
side note: FreeBSD documentation style guide says to not use contractions.  But of course we can have our own style.